fluentd/lib/generate_throttle_configs/generate_throttle_configs.gemspec

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+# can override for testing
+FLUENTD_VERSION = ENV['FLUENTD_VERSION'] || "0.12.0"
+
+Gem::Specification.new do |gem|
+  gem.name          = "generate_throttle_configs"
+  gem.version       = "0.0.1"
+  gem.authors       = ["Jeff Cantrill"]
```

Comment:
These gemspec files aren't really to use to publish an actual gem, they are just there so that I can do 
```
GEM_HOME=vendor bundle install; GEM_HOME=vendor bundle exec rake test
```
i.e. developer workflow developing the tests.  Maybe there is a way to do this without an actual gemspec or with a skeleton gemspec, but I didn't think it really mattered in this case.